nuget/Microsoft.NETCore.Runtime.JIT.Tools/runtime.Linux.Microsoft.NETCore.Runtime.JIT.Tools.props

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <ItemGroup>
+    <File Include="$(_LLVMInstallDir)\bin\llvm-mca" TargetPath="runtimes\$(PackageTargetRuntime)\native\" />
```

Comment:
The main (mono) .props files include the .pdb/.dbg/.dwarf files (though when I downloaded one, I didn't see them, so I'm not sure what is actually going on).  However, the objwriter ones do not, so I'm not sure what the convention is to follow here.

Reply:
It is probably fine - I think the reason not to include them is because of the package size. When looking at the `.pdb` file for `llvm-mca`, it was 60mb; `FileCheck.pdb` was 20mb.